go/exorelay/exorelay_test.go

```diff
@@ -130,6 +167,56 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^I register a handler for the "([^"]*)" message$`, func(messageName string) error {
+		mapping := map[string]func(map[string]interface{}){}
```

Comment:
Correction, I see that in your next PR you define a MessagePayload. That's the right approach!